stories/18-Avatar.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { MLAvatar } from 'marklogic-ui-library'
+import { text, withKnobs } from '@storybook/addon-knobs'
+
+export default {
+  title: 'Data Display/MLAvatar',
+  component: MLAvatar,
+  decorators: [withKnobs],
+}
+
+export const Basic = () => <MLAvatar size={text('size')} />
+
+export const WithInitials = () => (
+  <MLAvatar size={text('size')}>
+    {text('childText', 'DHS')}
+  </MLAvatar>
+)
+export const WithImage = () => (
+  <MLAvatar
+    size={text('size')}
+    src={text('src', 'https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png')}
```

Comment:
`text` knobs should have a second argument for the default string. I think this isn't the kind of knob you want for this, anyway — it should be one of a few options, or a number (or rather, a combo of those two options somehow)

Reply:
I was able to specify the union of number and limited text options in PropTypes, but I don't know if Knobs can do it. Entering a number does change the size but I now see it does so in an erroneous way. I'll see if there's a better knob.

Reply:
OK, there is no built in knob, but I added integer parsing.